### ISSUE TITLE
Add option to configure version of the DEMIS adapter which is deployed

### ISSUE THREAD

**Issue by vidi42** [hzi-braunschweig/SORMAS-Docker#226]
### Current situation
At the moment the DEMIS adapter version is hardcoded in the `/sormas/start-server.sh` script.
```echo -e "\ninterface.demis.jndiName=java:global/sormas-demis-adapter-1.4.1/DemisExternalLabResultsFacade" >>${DOMAIN_DIR}/sormas.properties```

However on some test environments we would like to always deploy the latest version of DEMIS which requires us a more flexible way to configure the DEMIS adapter version.
The workaround we use is to manually change the `start-server.sh` script and use the following line instead of the one above
```echo -e "\ninterface.demis.jndiName=java:global/sormas-demis-adapter/DemisExternalLabResultsFacade" >>${DOMAIN_DIR}/sormas.properties```

### Feature description
As a suggestion the version of the DEMIS adapter can be parametrized and use the `.env` variable to supply it.

Have a new variable: `DEMIS_ADAPTER_VERSION` which is either filled with a fixed version or not and tweak the `star-server.sh` script to have something like:

```
if [ ! -z "$DEMIS_ADAPTER_VERSION" ];then
echo -e "\ninterface.demis.jndiName=java:global/sormas-demis-adapter-${DEMIS_ADAPTER_VERSION}/DemisExternalLabResultsFacade" >>${DOMAIN_DIR}/sormas.properties
else 
echo -e "\ninterface.demis.jndiName=java:global/sormas-demis-adapter/DemisExternalLabResultsFacade" >>${DOMAIN_DIR}/sormas.properties
fi
```

**Comment by nliakm** [comment]
It looks like this option was already implemented in the latest release candidate (#229) and should be available to use.
However that change was not really documented but its now possible to use `DEMIS_VERSION` as an environment passed to the SORMAS service.

**Comment by vidi42** [comment]
We can work with this approach. Tested
